website/components/learn-nomad/style.css

```diff
@@ -0,0 +1,151 @@
+.g-learn-nomad {
+  padding-top: 88px;
+  padding-bottom: 88px;
+  background-image: url(/img/nomad-panel-learn.svg);
+  background-size: contain;
+  background-position: bottom right;
+  background-repeat: no-repeat;
+
+  @media (max-width: 768px) {
+    padding-top: 64px;
+    padding-bottom: 64px;
+  }
+
+  & .g-grid-container {
```

Comment:
This overload of an existing global style is kind of a bummer. If it's strictly necessary, that's ok but something we should be trying to avoid. If it's needed, worth putting a comment here that provides some context.

Reply:
Yeah, this is a good callout - to be honest we shouldn't be using the `g-grid-container` if we are doing this.  This definitely sneaked past our early review process.

I'll file a ticket here to get this addressed in the future, but I think we'll hold off for this PR.

Reply:
Resolving this here, created a ticket for this to be handled in the future.